### DISCUSSION TITLE
Exclude filter for Update Model tab

### DISCUSSION THREAD

**Discussion by silveroxides** [BlafKing/sd-civitai-browser-plus#242] [Ideas]
Right now it would take me about 4 hours to do any updates among all my models because it goes through all of them looking to see if they are present on civitai. Thing is I merge and train models regularily and test them in a1111 and I have no need for the extension to even touch these models because they don't exist anywhere but on my local machine.
So a way to filter out models that should not be checked feels like something that would save a lot of time.

**Comment by BlafKing** [comment]
Sounds good!

I'll think I implement this by letting you select one ore more folders to exclude from scanning in the settings, additionally I'll make it so that models that weren't found on civitAI will not be scanned again, (I'll also add an extra setting to override this behaviour if ever required)

Thanks for the suggestion!
